governance/xc_admin/packages/xc_admin_frontend/contexts/PythContext.tsx

```diff
@@ -46,15 +51,32 @@ export const PythContextProvider: React.FC<PythContextProviderProps> = ({
   multisigSignerKeyToNameMapping,
 }) => {
   const { isLoading, connection, rawConfig } = usePyth()
+  const [lazerState, setLazerState] = useState<LazerState | null>(null)
   const [
     productAccountKeyToSymbolMapping,
     setProductAccountKeyToSymbolMapping,
   ] = useState<AccountKeyToSymbol>({})
   const [priceAccountKeyToSymbolMapping, setPriceAccountKeyToSymbolMapping] =
     useState<AccountKeyToSymbol>({})
 
+  // Fetch Lazer state
   useEffect(() => {
-    if (!isLoading) {
+    const fetchLazerState = async () => {
+      try {
+        // Get the Lazer config and transform it to downloadable format
+        const lazerConfig = getConfig[ProgramType.PYTH_LAZER]({})
+        // Since we know this is a LazerState object, we can cast it
+        setLazerState(lazerConfig.state)
+      } catch (error) {
+        console.error('Error fetching Lazer state:', error)
+      }
+    }
+
+    fetchLazerState()
+  }, [])
```

Comment:
I'm a bit confused why this is async, it looks like everything it does is sync?

Also, you might like this pattern which I always use for client-side data fetching: https://github.com/pyth-network/pyth-crosschain/blob/main/packages/component-library/src/useData/index.ts.  It has a few nice things:

- This uses `swr` so you can easily configure it to automatically revalidate data if you so desire
- It returns a tagged union of states so handling loading / error / before loading states is not only trivial, but typescript will actually enforce that you don't forget to do it

Here's an example of recent code I wrote that uses it: https://github.com/pyth-network/pyth-crosschain/blob/main/apps/entropy-explorer/src/components/Home/results.tsx#L31-L62